console/frontend/src/main/frontend/js/app/services/sweetalert.service.js

```diff
@@ -39,26 +40,36 @@ appModule.service('SweetAlert', ['Debug', function (Debug) {
 			showCancelButton: true,
 		};
 		angular.merge(options, options, this.defaults.apply(this, arguments));
+		if (!options.callback)
+			return swal(options);
 		return swal(options, options.callback);
```

Comment:
Het is misschien een kwestie van stijl en voorkeuren, ik zou zelf kiezen voor een ternary statement hier:
```
return options.callback 
    ? swal(options, callback)
    : swal(options)
```

Doet in principe het zelfde, maar leest naar mijn mening iets fijner.